README.md

```diff
@@ -8,11 +8,13 @@ Formidable Labs flavored React component boilerplate. This is a set of opinions
 on how to start a React component.
```

Comment:
GLOBAL: Follow existing line breaking / wrapping conventions. (Think it's at 80 now, but 100 is probably ok -- or whatever lint is set at).